test/JDBC/expected/SELECT_INTO_TEST-vu-verify.out

```diff
@@ -0,0 +1,300 @@
+
```

Comment:
why aren't we testing any sys or information_schema_tsql views anywhere? How are we making sure that sys.column (for example) is showing correct details